### DISCUSSION TITLE
Q: 引入captcha时没法注册和修改密码（500错误） || Q: Unable to register and change password when captcha is introduced (500 error)

### DISCUSSION THREAD

**Discussion by SinzMise** [walinejs/waline#2005] [Q&A] [answered]
### Describe the bug | 问题描述

之前引入recaptcha之后就会出现500
![O~6HMQWR98REWI1UC5MJ](https://cdn.staticaly.com/gh/SinzMise/picx-images-hosting@master/20230611/O~6HMQWR98REWI1UC5MJ[1T.bem0iamum1c.webp)
删掉recaptcha相关变量就正常了
（目前问题只出现在“重置密码”）
### Website URL | 问题网站

https://waline.comment.sinzmise.eu.org/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

500 will appear after recaptcha was introduced before
![O~6HMQWR98REWI1UC5MJ](https://cdn.staticaly.com/gh/SinzMise/picx-images-hosting@master/20230611/O~6HMQWR98REWI1UC5MJ[1T.bem0iamum1c.webp)
It is normal to delete the recaptcha related variables
(Currently the problem only occurs in "Reset Password")
### Website URL | Problem Website

https://waline.comment.sinzmise.eu.org/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud (https://leancloud.app)

**Comment by SinzMise** [comment]
最新提醒：Turnstile也有这个问题，而且注册页面也受到了影响（那个用的Netlify部署）
问题出现在：https://wle.pl.sinzmise.gq/.netlify/functions/comment/

---
Latest reminder: Turnstile also has this problem, and the registration page is also affected (that uses Netlify deployment)
The problem appears here: https://wle.pl.sinzmise.gq/.netlify/functions/comment/

**Comment by lizheming** [comment, marked as answer]
同 https://github.com/walinejs/waline/issues/2024 已修复
